docs-kits/kits/Connector Kit/Adoption View/00_adoption-view.md

```diff
@@ -1,13 +1,4 @@
----
-id: Adoption View
-title: Adoption View
-description: 'Connector Kit'
-sidebar_position: 1
----
-
-![Connector kit banner](@site/static/img/ConnectorKitIcon.png)
-
-## Connector KIT
+# Adoption View
```

Comment:
The banner icon is still needed

Reply:
I'll move it below the heading, cause an .md file has to start with a heading.